### PR TITLE
Added name to Account

### PR DESCRIPTION
**Pull request by rdgoite**
Updated Account registration to record User's name so the entries are easily identifiable. This is part of initial implementation plans of manually setting roles for the Wranglers.

### CODE REVIEW

File: src/main/java/org/humancellatlas/ingest/security/common/jwk/JwtVerifierResolver.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    private final Optional<String> issuer;
+
+    public JwtVerifierResolver(JwkVault jwkVault, String audience, String issuer) {
+        this.jwkVault = jwkVault;
+        this.audience = Optional.ofNullable(audience);
+        this.issuer = Optional.ofNullable(issuer);
+    }
+
+    public String getIssuer() {
+        return issuer.get();
```
Comment (rolando-ebi) [inline src/main/java/org/humancellatlas/ingest/security/common/jwk/JwtVerifierResolver.java:31]: Is it best to just return the Optional issuer here? .get() will throw an exception if issue isn't present.

File: src/test/java/org/humancellatlas/ingest/security/AccountServiceTest.java [hunk 2]
```diff
@@ -54,7 +59,9 @@ void success() {
 
             //and:
             var savedAccount = accountCaptor.getValue();
-            assertThat(savedAccount.getProviderReference()).isEqualTo(providerReference);
+            assertThat(savedAccount)
+                    .extracting("providerReference", "name")
```
Comment (rolando-ebi) [inline src/test/java/org/humancellatlas/ingest/security/AccountServiceTest.java:63]: This breaks encapsulation of the Account class, and is brittle if renaming the provider/name fields. Assertions on the return values of the respective getters instead would address this

Comment (rolando-ebi) [reply src/test/java/org/humancellatlas/ingest/security/AccountServiceTest.java:63]: `.extracting(Account::getProviderReference, Account::getName)`

File: src/main/java/org/humancellatlas/ingest/security/common/jwk/JwtVerifierResolver.java (hunk 1 above)
Comment (rdgoite) [reply src/main/java/org/humancellatlas/ingest/security/common/jwk/JwtVerifierResolver.java:31]: The Optional is used here mostly to sanitise internal implementation. The getter remains pretty much nullable as it was before.

**Pull request merged**